load-test-framework/src/main/proto/loadtest.proto

```diff
@@ -16,6 +16,14 @@ service Loadtest {
   rpc Check(CheckRequest) returns (CheckResponse);
 }
 
+service Adapter {
```

Comment:
So this service itself is not an "adapter" -- the adapter part is the AdapterTask which runs in Java makes an rpc to this service.
The service here is more of a 'slave' or 'worker'. Suggest calling this LoadtestSlave, LoadtestWorker, or just Worker. I prefer LoadtestWorker, but any of those is fine.